react/components/FixedButton.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import ReactDOM from 'react-dom'
+
+class FixedButton extends Component {
+  constructor(props) {
+    super(props)
+
+    this.ref = React.createRef()
+    this.state = { mounted: false }
+    this.el = document.createElement('div')
+  }
+
+  componentDidMount() {
+    this.fixedRoot = this.ref.current
+
+    while (this.fixedRoot &&
+          window.getComputedStyle(this.fixedRoot)
+            .getPropertyValue('overflow') !== 'auto') { this.fixedRoot = this.fixedRoot.parentElement }
+
+    if (!this.fixedRoot)this.fixedRoot = document.body
+
+    this.fixedRoot && this.fixedRoot.appendChild(this.el)
+    this.setState({ mounted: true })
+  }
+
+  componentWillUnmount() {
+    this.fixedRoot && this.fixedRoot.removeChild(this.el)
+  }
+
+  render() {
+    const { mounted } = this.state
+
+    if (!mounted) return <div ref={this.ref} />
+
+    const children = this.props.children
+
+    return ReactDOM.createPortal(
+      <Fragment>
+        <div className="o-0">
+          {children}
```

Comment:
Why?

Reply:
To 'simulate' space at the end of the file, otherwise, the button will cover something at the footer

Reply:
I mean, why is the children rendered two times?

Reply:
One time just to add space at the end of the file and the second time for the fixed button.

Reply:
I talked to @lbebber, he said to add a tabIndex="-1"